samples/tab-channel-group-config-page-auth/csharp/ConfigTabAuthentication/Views/Authentication/AuthStart.cshtml

```diff
@@ -29,42 +29,26 @@
 </head>
 <body>
     <script src="https://statics.teams.cdn.office.net/sdk/v1.6.0/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.15/js/adal.min.js" crossorigin="anonymous"></script>
+    <script src="https://alcdn.msauth.net/browser/2.24.0/js/msal-browser.min.js" crossorigin="anonymous"></script>
     <script type="text/javascript">
         microsoftTeams.initialize();
 
         // Get the tab context, and use the information to navigate to Azure AD login page
         microsoftTeams.getContext(function (context) {
-            // ADAL.js configuration
-            let config = {
+
+        let msalConfig = {
+            auth: {
                 clientId: '@ViewBag.AuthStartClientId',
                 redirectUri: window.location.origin + "/AuthEnd",       // This should be in the list of redirect uris for the AAD app
-                cacheLocation: "localStorage",
                 navigateToLoginRequestUrl: false,
-            };
-
-            // Setup extra query parameters for ADAL
-            // - openid and profile scope adds profile information to the id_token
-            // - login_hint provides the expected user name
-            if (context.upn) {
-                config.extraQueryParameters = "scope=openid+profile&login_hint=" + encodeURIComponent(context.upn);
-            } else {
-                config.extraQueryParameters = "scope=openid+profile";
-            }
-
-            // Use a custom displayCall function to add extra query parameters to the url before navigating to it
-            config.displayCall = function (urlNavigate) {
-                if (urlNavigate) {
-                    if (config.extraQueryParameters) {
-                        urlNavigate += "&" + config.extraQueryParameters;
-                    }
-                    window.location.replace(urlNavigate);
```

Comment:
Are we not passing custom query parameters in MSAL as we are passing those in ADAL?

Reply:
No, Working without this parameters